tools/RAiDER/models/weatherModel.py

```diff
@@ -324,7 +324,10 @@ def _get_heights(self, lats, geo_hgt, geo_ht_fill=np.nan):
         Transform geo heights to WGS84 ellipsoidal heights
         '''
         geo_ht_fix = np.where(geo_hgt != geo_ht_fill, geo_hgt, np.nan)
-        lats_full = np.broadcast_to(lats[...,np.newaxis], geo_ht_fix.shape)
+        print ('LATS SHAPE', lats.shape)
```

Comment:
some diagnostics to show the shapes; HRRR conus (which works) and ERA5 are aligned dimension wise but the latter doesn't work